xocto/ranges.py

```diff
@@ -798,7 +798,7 @@ def days(self) -> int:
         """
         Return the number of days between the start and end of the range.
         """
-        return (self.end - self.start).days
+        return (self.end - self.start).days + 1
```

Comment:
Thought: This might be one to use the experiments framework on in Kraken. We would be able to observe failures in datadog without causing issues, and correct them as found.

Reply:
I'm open to that. That's a problem for step two though (upgrading the xocto version in Kraken). Priority for now is just getting the change accepted here.

Reply:
It might be worth adding an old-and-new function here for the transition, so that you can test them side-by-side, otherwise the upgrade will get stuck behind this change.

Reply:
Thinking about it, I don't think that this is a good candidate for the experiments framework after all, because that's designed around cases where you want both codepaths to have exactly the same result, but we know that we're making a change here. Sorry for the confusion!